fluent-plugin-protobuf/lib/fluent/plugin/parser_protobuf.rb

```diff
@@ -4,6 +4,8 @@
 require_relative '../../types_pb'
 require_relative '../../remote_pb'
 
+WriteRequest = Prometheus::WriteRequest
```

Comment:
Do you really need this change?

Reply:
Yes, otherwise our `module Prometheus` conflicts with the fluentd-prometheus plugin's `module Prometheus` and our parsing breaks.